apps/api/src/domain/repositories/interfaces/IConnectionRepositoryForResource.ts

```diff
@@ -0,0 +1,23 @@
+// Update this
```

Comment:
This isn't used. I suppose it's ok to keep this for now, in case we go forward with the `for particular resource` abstraction. But if we don't do that soon, remove this unused interface.